### ISSUE TITLE
Part 3, "CQL Functions"... how to link to the functions resource?

### ISSUE THREAD

**Issue by aaime** [opengeospatial/ogcapi-features#965]
When a server implements functions, it "SHALL support the HTTP GET operation at the /functions path."

Now, all resources are linked and this one should be linked from the landing page. But I cannot find a "rel" that should be used to link to functions.

In a similar way to how queryables are linked using the ``http://www.opengis.net/def/rel/ogc/1.0/queryables`` rel, I would expect to find a ``http://www.opengis.net/def/rel/ogc/1.0/functions`` rel for the functions resource... but the specification does not seem to contain one.

**Comment by cportele** [comment]
Good point, I agree. It seem we simply forgot about the link. We should add a recommendation for such a link (it cannot be a requirement as it would be a breaking change).
